util/rlp/fuzz/fuzz_targets/append_raw.rs

```diff
@@ -0,0 +1,19 @@
+#![no_main]
+#[macro_use] extern crate libfuzzer_sys;
+#[cfg(feature = "nightly")]
```

Comment:
I'm having trouble getting features to work nicely with cargo-fuzz. For example, to enable the feature, one must pass the `--features <feature-name` flag to `cargo build`.

I haven't been able to find documentation on how to enable a feature from the build script, or through cargo-fuzz options. The entire project compiles fine with this feature inclusion, but now the fuzz tests will not run with `cargo fuzz run untrusted`.

Currently I am trying to require the nighlty feature to build the binary targets with `required-feature` option under `parity/util/rlp/fuzz/Cargo.toml`. This stops the binaries from being built, but it is unclear how to enable these features when using a nightly compiler.